backend/logic/products/index.js

```diff
@@ -358,6 +399,13 @@ async function deleteProduct(shop, productId) {
   writeProductsFile(shop, updatedProducts)
   await removeProductData(shop, productId)
 
+  await Product.destroy({
```

Comment:
What happens if there is a pending order for the item (ex: due to the use of a PayPal echeck) - will that cause any issue processing the order?

Rather than deleting the row, an alternative would be to have a status column with enum values Active/Inactive.

Reply:
Gotcha, will add it

Reply:
Have more questions regarding this, even if we add the status column, a user could still buy an "Inactive" item. Also, the productId conflicts (in a shopId) could cause some weird things since `Product.findOne` is used.